src/WCSimWCTrigger.cc

```diff
@@ -97,6 +98,9 @@ void WCSimWCTriggerBase::GetVariables()
 int WCSimWCTriggerBase::GetPreTriggerWindow(TriggerType_t t)
 {
   switch(t) {
+  case kTriggerNoTrig:
+    return 0;
```

Comment:
Can you can this to:
`return -WCSimWCTriggerBase::LongTime;`
(the purpose of the trigger is to save everything, not just digits with positive times)

Reply:
done